datasync/utils.go

```diff
@@ -0,0 +1,28 @@
+package datasync
+
+import (
+	"crypto/ecdsa"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/vacp2p/mvds/state"
+)
+
+func ToGroupID(data []byte) state.GroupID {
+	g := state.GroupID{}
+	copy(g[:], data[:])
+	return g
+}
+
+// ToOneToOneGroupID returns a groupID for a onetoonechat, which is taken by
+// concatenating the bytes of the compressed keys, in ascending order by X
```

Comment:
Would it make sense to add a link to the relevant part of the spec for this? For future reference